### PR TITLE
Move getPool to local router from edge

### PR DESCRIPTION
**Pull request by p0mvn**
## What is the purpose of the change:

Given that we rely on SQS pools, we can leverage the SQS regional co-location with the FE. As a result, we begin moving the pools router from edge to local.

This move will also allow us to eventually reduce complexity of some of the complex pool tRPCs.

For scope limitation reasons, we move tRPC by tRPC as to minimize the impact of accidental regressions..

### CODE REVIEW

**Comment by jonator** [comment]
If we branch from `roman/switch-aprs-sidecar` perhaps we'll have less conflicts?

**Comment by p0mvn** [comment]
> If we branch from `roman/switch-aprs-sidecar` perhaps we'll have less conflicts?

I think we should QA that change before merging. However, this can be merged without QA upon approval.

I propose not blocking this PR on QA of the APRs - wdyt?

I will resolve conflicts

**Comment by jonator** [comment]
@p0mvn we renamed the getMarketIncentivePools procedure to just getPools once that data became available in SQS. We'll prob want to migrate that to local as well. IMO this PR is low prio enough to allow it to be blocked by that PR as it gets QA'd, it's in the same class of changes anyways.

File: packages/trpc/src/pools.ts
Comment (p0mvn) [inline packages/trpc/src/pools.ts:151]: Had to move `getSharePool` as well since it depended on `getPool`

**Comment by p0mvn** [comment]
> @p0mvn we renamed the getMarketIncentivePools procedure to just getPools once that data became available in SQS. We'll prob want to migrate that to local as well. IMO this PR is low prio enough to allow it to be blocked by that PR as it gets QA'd, it's in the same class of changes anyways.

We merged the APRs so this should be unblocked now

**Review by JoseRFelix — APPROVED** [review]

**Comment by JoseRFelix** [comment]
@p0mvn Is this PR still relevant?

**Review by jonator — APPROVED** [review]

**Pull request closed without merging**
